src/components/AccountWidget/AccountWidget.jsx

```diff
@@ -63,19 +63,24 @@ const mapStateToProps = (state) => {
   };
 };
 
+function RequestLayer(props) {
```

Comment:
it looks like this is commented out so you can probably just remove it.

Reply:
nvm, I misread the diff

Reply:
but actually you can remove the part that's commented out after the diff

Reply:
I thought it may have been structured that way to more easily make some planned change to componentWillUpdate, so left it in. I'll go ahead and remove it